lib/omnisharp-atom/features/signature-help.ts

```diff
@@ -0,0 +1,31 @@
+import ClientManager = require('../../omni-sharp-server/client-manager');
+import OmniSharpAtom = require('../omnisharp-atom');
+import Omni = require('../../omni-sharp-server/omni')
+
+class SignatureHelp {
+
+    constructor(private atomSharper: typeof OmniSharpAtom) {
```

Comment:
Ah k, I'd seen this used elsewhere. Should also update the Code Action bits as it's the same pattern.